test/acceptance/testing.go

```diff
@@ -203,6 +210,19 @@ func (c *AccTestCase) terraformApply() error {
 }
 
 func (c *AccTestCase) terraformDestroy() error {
+	if c.RetryDestroy.NumberOfTry <= 0 {
```

Comment:
If we use `uint8` instead of `int` for `NumberOfTry` we can do a `== 0` check, it does not make sense to have a number of attempts < 0.

Reply:
retrier.ConstantBackoff( is waiting fo a int so that will need a cast. I just test like that to prevent a crash when initializing retrier but I can change it

Reply:
also goland does not forbid to put -1 in a uint so yeah it would be a utin8